### PR TITLE
[Merged by Bors] - chore(Mathlib/Tactic/ToAdditive/Frontend): isolate the `to_additive` attribute implementation in a new file `Tactic.ToAdditive.Frontend` (1/2)

### PR DESCRIPTION
**Pull request by Komyyy**
Ideally, we want to move the `attribute [to_additive]` commands in `Init.ZeroOne` into `Tactic.ToAdditive` without any other changes.
However, the `to_additive` attribute is initialized in the file and cannot be included in the same file.
So it would be necessary to isolate the original `to_additive` attribute implementation in a new file `Tactic.ToAdditive.Frontend`, and create an new file `Tactic.ToAdditive` as a placeholder for the `attribute [to_additive]` commands in `Init.ZeroOne`.
But if we put these two changes in the same PR, git cannot recognizes that `Tactic.ToAdditive` is renamed, so we split the PR.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by Komyyy** [comment]
@jcommelin
Please delegate me.
In #14750, I was already delegated, and that PR should be merged after this PR is merged immediately.
If you delegate me, I'm going to merge these two PRs quickly.

**Review by YaelDillies — APPROVED** [review]
maintainer merge

**Comment by jcommelin** [comment]
bors d+

bors merge

**Pull request closed without merging**
